hasura.planx.uk/migrations/1706201208594_alter_table_public_team_integrations_add_column_staging_bops_secret/up.sql

```diff
@@ -0,0 +1,9 @@
+alter table "public"."team_integrations" add column "staging_bops_secret" text
+ null;
+
+comment on column "public"."team_integrations"."staging_bops_secret" is E'Secret required for BOPS submission in the format <API_KEY>:<INITIALIZATION_VECTOR>';
+
+alter table "public"."team_integrations" add column "production_bops_secret" text
+ null;
+
+comment on column "public"."team_integrations"."production_bops_secret" is E'Secret required for BOPS submission in the format <API_KEY>:<INITIALIZATION_VECTOR>';
```

Comment:
Helpful reminder :+1:

Reply:
I actually have a small PR for this from the previous creation of `team_integrations` https://github.com/theopensystemslab/planx-team-onboarding/pull/13/files

I'll pick up the remaining updates shortly also 👍

Reply:
Next PR here - https://github.com/theopensystemslab/planx-team-onboarding/pull/14